ui/src/shared/components/CodeSnippet.scss

```diff
@@ -4,27 +4,32 @@
 */
 
 .code-snippet {
-  margin: $ix-marg-a 0;
-  border-radius: $ix-radius;
+  border-radius: $cf-radius;
   overflow: hidden;
+  border: $cf-border solid $g2-kevlar;
+  background-color: $g2-kevlar;
+  margin-bottom: $cf-marg-b;
 }
 
 .code-snippet--scroll {
   min-height: 26px !important;
-  background-color: $g4-onyx;
+  max-height: 200px;
+  border-radius: $cf-radius-sm;
+  overflow: hidden;
 }
 
 .code-snippet--text {
   font-size: 13px;
```

Comment:
is there a font variable we should be using here?

Reply:
Yes! Good looking out. We have one in the clockface variables